### PR TITLE
Make structure analysis work on unit test with return inside of if statement

### PR DESCRIPTION
**Pull request by ptomin**
- Create structure analysis unit test with return inside of if statement 
- Virtualize return edges before last resort refinement

### CODE REVIEW

File: src/Decompiler/Structure/Region.cs [hunk 1]
```diff
@@ -47,6 +47,17 @@ public class Region
         public List<AbsynStatement> Statements { get; set; }
         public Expression Expression { get; set; }
 
+        /// <summary>
+        /// Check if region contains only return statement
+        /// </summary>
```
Comment (uxmal) [inline src/Decompiler/Structure/Region.cs]: Thanks. I'd also change the comment to make it extra clear "Return true if region consists of a single AbsynReturn statement."

Comment (ptomin) [reply src/Decompiler/Structure/Region.cs]: Done. I have amended this commit one more time.

**Comment by uxmal** [comment]
Looks good now. Let's pull it.

**Comment by uxmal** [comment]
Travis-CI builds are starting to exceed some 10 minute boundary. I woudn't be too concerned; the AppVeyor build is fine. I'll go ahead and pull now.

**Pull request merged**